src/query/interpret/eval.hpp

```diff
@@ -467,7 +469,102 @@ class ExpressionEvaluator : public ExpressionVisitor<TypedValue> {
         throw QueryRuntimeException("Invalid property name {} for Graph", prop_name);
       }
       default:
-        throw QueryRuntimeException("Only nodes, edges, maps and temporal types have properties to be looked-up.");
+        throw QueryRuntimeException(
+            "Only nodes, edges, maps, temporal types and graphs have properties to be looked up.");
+    }
+  }
+
+  TypedValue Visit(AllPropertiesLookup &all_properties_lookup) override {
+    TypedValue::TMap result(ctx_->memory);
+    std::map<std::string, TypedValue> result_;
```

Comment:
@antepusic can you actually make this utils::pmr::unordered_map:

`utils::pmr::unordered_map<std::string, TypedValue> results(context.evaluation_context.memory);`

This way we will track memory

Reply:
Actually you can use utils::pmr::map directly, as we have for it constructors in TypedValue.

`utils::pmr::map<utils::pmr::string, TypedValue>`;